packages/core/src/internal.ts

```diff
@@ -0,0 +1,2 @@
+export * from './index';
```

Comment:
I'm not sure if it should export all of the core too.

Reply:
I agree

Reply:
me too

Reply:
@idoros should we?

Reply:
maybe we can skip the internal.ts and just import files directly from the internal folder

Reply:
I think it unifies the internal api in a way that can be simply mapped to package json exports when ts 4.5 will be out